### PR TITLE
Update services.yaml

### PR DESCRIPTION
**Pull request by narumigsa**
On the Products page, the logo for Notify.gov needs to be updated with this logo.

On the Products page, the link for Notify.gov needs to be updated to be this link: https://digital.gov/2023/02/07/collaborate-with-the-tts-public-benefits-studio/

On the Platforms page, the logo for Vote.gov needs to be updated with this logo

On the Platforms page, the link for FAC.gov needs to be updated to be: https://www.fac.gov/

-
-

## security considerations

none

### CODE REVIEW

File: _data/services.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ services:
   - name: Vote.gov
     link: https://vote.gov
     link_alt_text: link to Vote.gov homepage
-    logo: _img/vote-gov.jpg
+    logo: LOGO_votegov_2024.png
```
Comment (wesley-dean-gsa) [inline _data/services.yaml:68]: the build is saying this file doesn't exist

**Review by wesley-dean-gsa — CHANGES_REQUESTED** [review]

File: _data/services.yaml (hunk 1 above)
Comment (narumigsa) [reply _data/services.yaml:68]: what about now?

Comment (wesley-dean-gsa) [reply _data/services.yaml:68]: not yet... maybe `_img/LOGO_votegov_2024.png` ?

**Review by wesley-dean-gsa — APPROVED** [review]
nailed it!  :cake:

**Pull request merged**